server/container/guice/guice-common/src/test/java/org/apache/james/AbstractJamesServerTest.java

```diff
@@ -122,6 +125,7 @@ public void connectLMTPServerShouldSendShabangOnConnect() throws Exception {
 
     @Test
     public void connectJMAPServerShouldRespondBadRequest() throws Exception {
+        Assume.assumeTrue(server.getJmapPort().isPresent());
```

Comment:
I thought about that but now I separated Jmap and non-jmap testing so the assume is not need anymore. Server should be fail on start if non jmap port.